lib/index.js

```diff
@@ -106,6 +130,7 @@ exports.Model = class SchwiftyModel extends Model {
 
     parseValidationError(err) {
```

Comment:
If this wont require access to `this`, make it a `static`.  If it does require access to this, privatize it using `_` or `$` if that's appropriate in the context of objection.